pom.xml

```diff
@@ -857,6 +860,10 @@
             <groupId>commons-logging</groupId>
             <artifactId>commons-logging</artifactId>
           </exclusion>
+          <exclusion>
```

Comment:
Do we need to exclude netty? We have netty-bom in the dependency

Reply:
We are using `io.netty:netty-all:jar:4.1.107.Final`, this is older dependency.

Reply:
I don't follow. Since we have netty explicitly configured, it should pull the new netty, and there should be no need to exclude it

Reply:
Figured that netty is the old netty-all which is already deprecated, so it should be safe to exclude